src/components/Config/Config.tsx

```diff
@@ -2,13 +2,15 @@ import { useCallback, useMemo, useState } from 'react';
 import { Form, Modal, Button, Typography, Radio } from 'antd';
 import styled from 'styled-components';
 
-import { useAuth, useAutomateConnection } from '../../hooks';
 import { Network, ChainId, ConfirmationTime, ethereum } from '../../constants';
-import CopyInput from '../CopyInput';
+import { useAuth, useAutomateConnection } from '../../hooks';
 import { capitalizeFirstLetter } from '../../utils';
+import CopyInput from '../CopyInput';
 import PageTitle from '../PageTitle';
 import ConnectionSettings from './ConnectionSettings';
 
+const strategyPathKey = 'strategyPath';
```

Comment:
How is this better? You still have two `'strategyPath'` strings - here and in `StrategyList`. You are still repeating yourself and there's no benefit to what you did. At this point I'm not gonna explain what you need to do here. You must figure it out yourself

Reply:
i used constants file and exporting it from there, think its preventing declaring same variable in two files.